spec/services/articles/feed_spec.rb

```diff
@@ -12,16 +12,16 @@
   describe "#published_articles_by_tag" do
     let(:unpublished_article) { create(:article, published: false) }
     let(:tag) { "foo" }
-    let(:tagged_article) { create(:article, tags: [tag]) }
+    let(:tagged_article) { create(:article, tags: tag) }
```

Comment:
Is it fine for the tags to be a single object instead of an array?

Reply:
It's not only fine, it's CORRECT! The tags are actually a comma delimited string!